src/theme/TimelineComponent.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Timeline as TrivialTimeline, Event as Event } from 'react-trivial-timeline';
+
+export default function TimelineComponent({ children }: { children: Event }) {
+  return(
+    <TrivialTimeline lineColor="#ab5500">
```

Comment:
Wrapping this solely to set the a colour that works in both themes without everyone having to remember the colour in the markdown.